docs/examples/modules/DropdownExample/index.js

```diff
@@ -19,6 +20,11 @@ export default [
   {
     title: 'Types',
     examples: [
+      {
+        title: 'Dropdown Nested',
+        description: 'A dropdown with nested menu.',
+        component: DropdownNested,
+      },
```

Comment:
Nit: I'd move this one at the end of the array, to keep the docs as similar as possible to the original website